modules/lazyload/module.php

```diff
@@ -156,9 +184,31 @@ public function __construct() {
 		add_action( 'wp_footer', function() {
 			?>
 			<script type='text/javascript'>
+				const getReapetersBackgroundLazyload = ( lazyloadBackgrounds, reapterAttribute, dataAttribute) => {
+					const lazyloadRepeaterBackgrounds = document.querySelectorAll( `[${ reapterAttribute }]` );
+					const repeaterBackgrounds = [];
+
+					lazyloadRepeaterBackgrounds.forEach( ( repeater ) => {
+						const lazyloadSelector = repeater.getAttribute( dataAttribute );
+						if ( lazyloadSelector ) {
+							lazyloadBackground = repeater.querySelectorAll( lazyloadSelector );
+							if ( lazyloadBackground.length ) {
+								repeaterBackgrounds.push( ...lazyloadBackground );
+							}
+						}
+					} );
+					lazyloadBackgrounds = [ ...lazyloadBackgrounds, ...repeaterBackgrounds ];
+
+					return lazyloadBackgrounds;
+				};
+
 				const lazyloadRunObserver = () => {
 					const dataAttribute = 'data-e-bg-lazyload';
-					const lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded)` );
+					const reapterAttribute = 'data-e-bg-repeater';
+
+					let lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded, [${reapterAttribute}])` );
+					lazyloadBackgrounds = getReapetersBackgroundLazyload( lazyloadBackgrounds, reapterAttribute, dataAttribute );
```

Comment:
What is the different between repeaters and selectors?

Reply:
Repeaters are the item of repeater control.
The selector is CSS rule which apply to repeater control.